MIGRATION.md

```diff
@@ -11,10 +11,27 @@ There are several ways to create a client connection in `cloudant-node-sdk`:
 
 ## Other differences
 1. Using the `dotenv` package to store credentials in a file is not recommended. See the [external file configuration section](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration) in our API docs for handling this feature in our new library.
-1. Fetching the database object first before performing additional operations is not required. For example, in the case of updating a document you would first call `getDocument` to fetch and then `putDocument` to update.
+1.  In `cloudant-node-sdk` all operations are performed from the scope of the client instance and
+    not associated with any sub-scope like the database. There is no need to instantiate a
+    database object to interact with documents - the database name is included as part of
+    document operations. For example, in the case of updating a document you would first call
+    `getDocument({ db: dbName, docId: docId})` to fetch and then `putDocument({ 
+    db: dbName, docId: docId})` to update. As a result of which the `use`
+    operation also became obsoleted.
 1. Plugins are not supported, but several of the plugin features exist in the new library e.g. IAM, [automatic retries](https://github.com/IBM/ibm-cloud-sdk-common/#automatic-retries) for failed requests.
 1. Error handling is not transferable from `@cloudant/cloudant` to `@ibm-cloud/cloudant`. For more information go to the [Error handling section](https://cloud.ibm.com/apidocs/cloudant?code=node#error-handling) in our API docs.
-1. Custom HTTP client configurations in `@cloudant/cloudant` are not transferable to are not transferable to `@ibm-cloud/cloudant`. For more information go to the [Configuring the HTTP client section](https://github.com/IBM/ibm-cloud-sdk-common/#configuring-the-http-client) in the IBM Cloud SDK Common README.
+1. Custom HTTP client configurations in `@cloudant/cloudant` are not transferable to
```

Comment:
Well the Node.js `https.Agent` part _can_ be used, but it has to passed differently - I think that is covered by the linked doc anyway.

Reply:
Wdyt of 
```suggestion
1. Custom HTTP client configurations in `@cloudant/cloudant` can be set differently in
```

Reply:
Fixed in 2e65344